packages/cli/test/integration/runners/task-runner-process.test.ts

```diff
@@ -88,15 +88,16 @@ describe('TaskRunnerProcess', () => {
 		// @ts-expect-error private property
 		runnerProcess.process?.kill('SIGKILL');
 
-		// Assert
-		// Wait until the runner is running again
-		await retryUntil(() => expect(runnerProcess.isRunning).toBeTruthy());
-		expect(runnerProcess.pid).not.toBe(processId);
+		// Wait until the runner has exited
+		await runnerProcess.runPromise;
 
+		// Assert
 		// Wait until the runner has connected again
 		await retryUntil(() => expect(getNumConnectedRunners()).toBe(1));
 		expect(getNumConnectedRunners()).toBe(1);
 		expect(getNumRegisteredRunners()).toBe(1);
+		expect(runnerProcess.pid).toBeDefined();
```

Comment:
these two assertions seem redundant. Just one of these should be enough, right?

Reply:
True true. Removed it